packages/create-waku/src/index.ts

```diff
@@ -202,15 +204,40 @@ async function init() {
     await installTemplate(root, packageName, templateRoot, templateName);
   }
 
-  // TODO automatically installing dependencies
   // 1. check packageManager
   // 2. and then install dependencies
-
-  console.log(`\nDone. Now run:\n`);
-  console.log(`${bold(green(`cd ${targetDir}`))}`);
-  console.log(`${bold(green(commands.install))}`);
-  console.log(`${bold(green(commands.dev))}`);
   console.log();
+  console.log(`Installing dependencies by running ${commands.install}...`);
+
+  process.chdir(targetDir);
+
+  const installProcess = spawn(packageManager, ['install']);
+
+  installProcess.stdout.setEncoding('utf8');
+  installProcess.stdout.on('data', (data) => {
+    console.log(data);
```

Comment:
https://github.com/dai-shi/waku/pull/728#discussion_r1620085498

Does it show the indicator as expected? cc @ojj1123

Reply:
Yes. I could see the indicator as expected!

Reply:
[The docs is explaining a difference between `stdio: "inherit"` and `stdio: "pipe"`](https://nodejs.org/api/child_process.html#optionsstdio) but it's a bit confusing for me.
I've tried to use `stdio: "inherit"`, it could show the loading indicator.

### `stdio: "pipe"`(default)
- child stdios(stdin, stdout, stderr) and parent ones are mapped(piping) each other. But if we want to access the data, we have to use the events like this:
```js
childProcess.stdout.on("data", (data) => {
  console.log(data)
})
```

### `stdio: "inherit"`
- parent's stdios are handed off to the child process. That is, child process could use parent's stdios. **So we don't need to use the events.**

I think no matter we use `stdio: "pipe"` or `stdio: "inherit"`. But if we use "inherit", we don't need to use the events:

```diff
diff --git a/packages/create-waku/src/index.ts b/packages/create-waku/src/index.ts
index 1eb01d3..98ede50 100644
--- a/packages/create-waku/src/index.ts
+++ b/packages/create-waku/src/index.ts
@@ -210,16 +210,8 @@ async function init() {

   process.chdir(targetDir);

-  const installProcess = spawn(packageManager, ['install']);
-
-  installProcess.stdout.setEncoding('utf8');
-  installProcess.stdout.on('data', (data) => {
-    console.log(data);
-  });
-
-  installProcess.stderr.setEncoding('utf8');
-  installProcess.stderr.on('data', (data) => {
-    console.log(data);
+  const installProcess = spawn(packageManager, ['install'], {
+    stdio: 'inherit',
   });

   installProcess.on('close', (code) => {
```

Reply:
+) If we use `stdio: "inherit"`(or anything other than "pipe"), childProcess's `stdin`, `stdout` and `stderr` could be `null`:

![스크린샷 2024-07-01 19 23 18](https://github.com/dai-shi/waku/assets/33178048/7f8a597d-9590-4ccc-8d66-f29bc5df3400)